components/settings/Settings.js

```diff
@@ -122,6 +135,60 @@ const SettingsDialog = React.createClass({
     cancelDialog() {
         this.closeDialog();
         this.props.onRequestClose();
+        this.setState({ allowImport: '', alertUser: undefined, modFolder: "", compileMod: false })
```

Comment:
you are right, I was thinking in importCellTemplate, where you can import more than ones, but there is not reason to import a model twice. It would make more sense to refresh the page and then to start a new model rather than import several models. Will remove